pyscriptjs/tests/integration/support.py

```diff
@@ -92,6 +93,32 @@ def init(self, request, tmpdir, http_server, logger, page):
 
     def init_page(self, page):
         self.page = page
+        # cache to store data via sha256(url)
+        cache = {}
+
+        # router to cache with fail 2x on requests
```

Comment:
I don't really understand what this comment means :)

Reply:
# cache storing request objects by a key computed as the sha256 of the url

Reply:
I am planning to also limit this to GET requests.